Content.Server/_White/Headcrab/HeadcrabComponent.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Content.Shared.Damage;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server._White.Headcrab;
+
+[Access(typeof(HeadcrabSystem))]
+[RegisterComponent]
+public sealed partial class HeadcrabComponent : Component
+{
+    /// <summary>
+    /// WorldTargetAction
+    /// </summary>
+    [DataField(required: true, customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+    public string JumpAction = "ActionHeadcrabJump";
+
+    [DataField]
+    public float ParalyzeTime = 3f;
```

Comment:
```suggestion
    public TimeSpan ParalyzeTime = TimeSpan.FromSeconds(3);
```
Use `TimeSpan` instead of float.